nutanix/data_source_nutanix_subnet_test.go

```diff
@@ -108,19 +105,14 @@ output "cluster" {
 
 resource "nutanix_subnet" "test" {
 	name = "dou_vlan0_test_%d"
-
-	cluster_reference = {
-	  kind = "cluster"
-	  uuid = "${data.nutanix_clusters.clusters.entities.0.metadata.uuid}"
-  }
-
+	cluster_uuid = "${data.nutanix_clusters.clusters.entities.0.metadata.uuid}"
 	vlan_id = %d
 	subnet_type = "VLAN"
 
 	prefix_length = 24
 	default_gateway_ip = "192.168.0.1"
 	subnet_ip = "192.168.0.0"
-	#ip_config_pool_list_ranges = ["192.168.0.5", "192.168.0.100"]
+	ip_config_pool_list_ranges = ["192.168.0.10 192.168.0.100"]
```

Comment:
not sure I fully understand why this is just a single string here? is it that each IP is part of the range?

Reply:
It is because this argument could accept different IP ranges, the API receive those ranges as an array of strings, so, to follow the same semantic we set this argument like this. 

We also added a validation in the schema.
https://github.com/terraform-providers/terraform-provider-nutanix/blob/dd96968d4e0a60ef4d49569fa77bdf49a42989c3/nutanix/resource_nutanix_subnet.go#L219